blockchain/tx_pool_test.go

```diff
@@ -181,7 +182,8 @@ func (c *testChain) State() (*state.StateDB, error) {
 // state reset and tests whether the pending state is in sync with the
 // block head event that initiated the resetState().
 func TestStateChangeDuringTransactionPoolReset(t *testing.T) {
-	t.Parallel()
```

Comment:
Revised the `t.Parallel` for keep consistency between parallel running tests in tx_pool_test.go